lib/Frontend/OutputFileMap.cpp

```diff
@@ -108,21 +100,42 @@ void OutputFileMap::dump(llvm::raw_ostream &os, bool Sort) const {
   }
 }
 
-bool OutputFileMap::parse(std::unique_ptr<llvm::MemoryBuffer> Buffer,
-                          StringRef workingDirectory) {
+void OutputFileMap::write(llvm::raw_ostream &os,
+                          ArrayRef<StringRef> inputs) const {
+  auto printQuotedString = [&os](StringRef s) -> void {
+    os << "\"" << s << "\"";
```

Comment:
Please use raw_ostream's `write_escaped` method for the string. (I get away with not doing that in some other places because the strings are all identifiers, but paths are not that nice.)

Reply:
Cool! Thanks.